stable/yugabyte/templates/service.yaml

```diff
@@ -186,6 +186,10 @@ spec:
       {{- if $root.Values.nodeSelector }}
       nodeSelector:
       {{ toYaml $root.Values.nodeSelector | indent 8 }}
+      {{- end }}
+      {{- if $root.Values.tolerations }}
+      tolerations:
+{{ toYaml $root.Values.tolerations | indent 8 }}
```

Comment:
This has to be unindented to make the `| indent 8` work, apparently. Unsure if there's a better way to do it.

This is actually also kind of an issue for `nodeSelector`, but I think it would only show up if there are more than one label in the selector. If there's only one, the selector is visually overindented but still valid YAML:

```yaml
    spec:
      nodeSelector:
        # should be here...
              # but is actually here
              foo: bar
```

Reply:
You are right. AFAIK better would be to have something like this,
```yaml
      {{- if $root.Values.tolerations }}
      tolerations:
      {{- toYaml $root.Values.tolerations | nindent 8 }}
      {{- end }}
```
The hyphen before `toYaml` will remove one line (along with our indentation). The `nindent` will add a new line and then indent our object.

Also, as you mentioned the nodeSelector needs to be fixed.